### PR TITLE
Enhance container_diff with rpm packages' comparison

### PR DESCRIPTION
**Pull request by b10n1k**
Add function to compare the rpm's version from the output of the
container-diff tool.
For convenience i picked the cpan::version tool to make the comparison.
Documentation https://metacpan.org/pod/distribution/version/lib/version.pm
Note the conversion as it might end to unexpected results in first glance.
For instance `version->parse('1.2')` returns `1.200` which will make a
comparison with 1.19 for example to evaluate the first bigger than the later.
To solve this and for more other reasons the v-string is required.

- Related ticket: https://progress.opensuse.org/issues/71050
- Verification run: http://aquarius.suse.cz/tests/4336

### CODE REVIEW

**Review by grisu48 — APPROVED** [review]
LGTM! It would be nice to have also verification runs for older versions, at least for 15-SP2 as well when updating existing test cases.

**Review by pdostal — APPROVED** [review]

**Comment by b10n1k** [comment]
i would like to run a VR in production before merging

File: lib/containers/container_images.pm
Comment (grisu48) [inline lib/containers/container_images.pm:169]: Wouldn't it be better to use the internals like

    die "$+{package} $stable_v is not updated to $updated_v" unless ($updated_v > $stable_v);

instead of including the `Test:Assert` module here? It's a very minor complain and both should be fine though.

Comment (b10n1k) [reply lib/containers/container_images.pm:169]: i like this. your suggestion is more perlish. Also i have minor changes i want to do. thus i am thinking to merge this and create an additional PR if there is no objection.

**Pull request merged**
